File: MSEProjects/EX6/models/game.js
import { Schema, model } from 'mongoose'

export default model(
    'Game',
    new Schema(
        {
            title: { type: String, required: true },
            price: { type: Number, required: true },
            description: { type: String, required: true },
            image: { type: String, required: true },
            quantity: { type: Number, required: true },
        },
        {
            timestamps: true,
        }
    )
)

File: Flutter/gstore/lib/data/game.dart
class Game {
  final String? _id;
  final String imagePath;
  final String title;
  final int price;
  final String description;
  final int qte;

  Game(this._id, this.imagePath, this.title, this.price, this.description,
      this.qte);

  // generate toString
  @override
  String toString() {
    return 'Game{path: $imagePath, name: $title, price: $price, description: $description, qte: $qte}';
  }
}
